contents/docs/session-replay/canvas-recording.mdx

```diff
@@ -21,3 +21,25 @@ Canvas elements are recorded at a rate of **4 frames per second** which may lead
     alt="Enable canvas recording in your PostHog" 
     classes="rounded"
 />
+
+## Canvas recording performance
+
+Canvas recording normally has no impact on your site, but it does need to capture the canvas to an image so can have performance impact on some sites.
+
+You can override the frequency the image is captured, and the quality of the captured image when initialising PostHog on your site
+
+```
+posthog.init('<ph_project_api_key>', {
+  api_host: '<ph_client_api_host>',
+  session_recording: {
+    captureCanvas: {
+      // accepts a number between 0 and 12
+      // if not set locally, when canvas recording is enabled, remote config sets this to 4
```

Comment:
it's best to comment on lines or files @mxzinke cos then we can thread :)
literally the worst UX feature of GitHub, that you can't thread on the conversation tab

default was a poor choice of words maybe... if you turn on canvas recoding, the backend provides a fixed value. to change it you have to override in config

Reply:
https://github.com/PostHog/posthog-js/pull/1721

Reply:
Thanks for clarifying. Now way more understandable.

It would be awesome if the docs on types on posthog-js repo would also have this information :)